screwdriver/screwdriver_test.go

```diff
@@ -77,6 +81,22 @@ func TestNew(t *testing.T) {
 		testMsg := "failed to get JWT: StatusCode 500"
 		assert.Equal(t, testMsg, err.Error())
 	})
+
+	t.Run("failure by makeURL", func(t *testing.T) {
+		testURL := "http://example.com:yyy"
+		testMsg := "failed to make request url:"
+		testToken := "token"
+		_, err := New(testURL, testToken)
+		assert.Contains(t, err.Error(), testMsg)
```

Comment:
Is it better to use `asser.Equal()` as [this](https://github.com/screwdriver-cd/sd-local/blob/5b14f95ce778ab0b775b48cce50fe25bdc979878/screwdriver/screwdriver_test.go#L159)?

Reply:
Fixed.